daml-foundations/daml-ghc/damldoc/src/DA/Daml/GHC/Damldoc/Render/Anchor.hs

```diff
@@ -27,7 +27,8 @@ import qualified Data.Char as C
 type Anchor = T.Text
 
 moduleAnchor :: Modulename -> Anchor
-moduleAnchor m = T.intercalate "-" ["module", convertModulename m, hashText m]
+-- calculating a hash on String instead of Data.Text as hash output of the later is different on Windows than other OSes
+moduleAnchor m = T.intercalate "-" ["module", convertModulename m, hashText $ show m]
```

Comment:
`show m` will wrap `m` in additional quotes, is `T.unpack` not sufficient?

Reply:
I don’t think you even need `T.unpack`. `Text` should be `Hashable` so `hashText m` should do the trick.

Reply:
Thanks @aherrmann-da, yes `T.unpack` is better, especially that `Modulename` is actually a `Text` as well.

@cocreature the problem is that hash of `Data.Text` is giving different results on Windows and Linux/Mac (see the PR description)

Reply:
Oh I see, sorry for the confusion.